styles/styles.css

```diff
@@ -421,3 +421,39 @@ ul.contributor-name li a {
 li.dev-name::marker {
   color: #007acc;
 }
+
+/*Media Query*/
+/*Responsive for tablets <770px*/
+@media screen and (max-width: 770px){
+  img.cardimg {
+    width: 45px;
+    height: auto;
+  }
+
+  .opponentcards::after{
+    content: "";
+    position: absolute;
+    background-image: url("../assets/imgs/back-cover/back-cover.png") !important;
```

Comment:
There's no need to include this since it has already been declared earlier. Near the line 179.

Reply:
Right, is this issue resolved or should I create another pull request?